tests/test_encode_decode.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
```

Comment:
There were more tests in the repo we're extracting the code from. These 2 seem to be written from scratch.

See: `tests/unit/test_base64dotted.py`. It's got what I believe is a good corner case coverage.